lib/install_shared.sh

```diff
@@ -344,7 +344,7 @@ function build_containers() {
 
     if [[ $build_docker_containers == true ]] ; then
         print_message "Building / Updating docker containers"
-        if docker info 2>/dev/null | grep rootless || [[ $(uname) == "Darwin" ]]; then
+        if docker info 2>/dev/null | grep rootless || docker info 2>/dev/null | grep lima-rancher-desktop || [[ $(uname) == "Darwin" ]]; then
```

Comment:
lima to my knowledge is a VM based solution. What is the rationale here to check for this support?

Reply:
is the sudo call failing on your system?

Reply:
Ok, understood. I guess it should be changed then from our side. The `sudo docker` is legacy and since we now have the first quirk with it we remove it. The check for lima feels very specific and going for a general requirement to have the docker command be non-root runnable seems like the better option here. Feel free to remove the line. Otherwise I will remove it after merge.